src/Service/ActivityPubManager.php

```diff
@@ -441,8 +442,10 @@ public function updateMagazine(string $actorUrl): ?Magazine
 
         if (isset($actor['endpoints']['sharedInbox']) || isset($actor['inbox'])) {
             if (isset($actor['summary'])) {
-                $converter = new HtmlConverter(['strip_tags' => true]);
-                $magazine->description = stripslashes($converter->convert($actor['summary']));
+                $magazine->description = $this->extractMarkdownSummary($actor);
+                if (!empty($rules)) {
+                    $magazine->rules = $rules;
+                }
```

Comment:
I might be missing it, is `$rules` still set here?

Reply:
no that part should be removed

Reply:
everything else in this PR seems fine to me, it's just removing these three lines I think otherwise it'll throw an undefined variable exception, unless I'm misreading it, difficult for me to be sure locally since I'm not sure how I could fake an AP event

Reply:
I don't think it will lead to exceptions because `empty` is null safe afaik, though yes ot should be removed.
If you want to, feel free to fix it and then merge it. I will not be able to 'til wednesday